flink-table/flink-table-planner-blink/src/test/java/org/apache/flink/table/sqlexec/SqlToOperationConverterTest.java

```diff
@@ -187,6 +192,45 @@ public void testCreateTableWithMinusInOptionKey() {
 		assertEquals(expected, sortedProperties.toString());
 	}
 
+	@Test
+	public void testCreateTableWithWatermark() throws FunctionAlreadyExistException, DatabaseNotExistException {
+		CatalogFunction cf = new CatalogFunctionImpl(
+			JavaUserDefinedScalarFunctions.JavaFunc5.class.getName(),
```

Comment:
Use `functionCatalog.registerTempSystemScalarFunction(` is ok, there is no need to create define a new function `JavaFunc5`. You can pick a function in `org.apache.flink.table.planner.expressions.utils.`

Reply:
This aims to test whether a catalog function name will be qualified correctly. Temporary system function will not be qualified.

Reply:
`JavaFunc5` is used in the last 2 commits.